packages/client/hmi-client/src/views/TA2Playground.vue

```diff
@@ -121,8 +126,10 @@ class SampleRenderer extends graphScaffolder.BasicRenderer<NodeData, EdgeData> {
 	}
 
 	renderNodes(selection: D3SelectionINode<NodeData>) {
-		const species = selection.filter((d) => d.data.type === 'species');
-		const transitions = selection.filter((d) => d.data.type === 'transition');
+		const species = selection.filter((d) => d.data.type === 'species' || d.data.type === 'S');
```

Comment:
We should use `NodeType.Species` here instead of the hardcoded string `'S'`.

When would we be using 'species' instead of 'S'? Can we consolidate these?

Reply:
'species' is used in some of the LotkaVolterra example - so is 'S'
Will definitely update to use the enum for this

Reply:
Sounds good. Hopefully we can consolidate going forward.